Gems/AtomLyIntegration/CommonFeatures/Code/Source/CoreLights/DirectionalLightComponentConfig.cpp

```diff
@@ -21,7 +21,7 @@ namespace AZ
             if (auto* serializeContext = azrtti_cast<SerializeContext*>(context))
             {
                 serializeContext->Class<DirectionalLightComponentConfig, ComponentConfig>()
-                    ->Version(7)
+                    ->Version(8)
```

Comment:
nitpick: FYI: When adding a new field, you don't actually need to bum the version on the serialize context. Anything with that is missing a field will just populate that field with the default value. The version bump is only needed for something more destructive like changing/removing a field, or introducing a version converter for modifying values in existing data.